### ISSUE TITLE
scripted agent upgrade fails on windows

### ISSUE THREAD

**Issue by andrewq** [scalyr/scalyr-agent-2#226]
```
C:\Program Files (x86)\Scalyr\bin>scalyr-agent-2-config.exe --upgrade-windows
2019-06-04 14:17:29.832Z ERROR [core] [scalyr_logging.pyc:441] [error="requestFailed"] Failed to send request due to exception.  Closing connection, will re-attempt :stack_trace:
  Traceback (most recent call last):
    File "scalyr_agent\scalyr_client.pyc", line 265, in __send_request
    File "scalyr_agent\connection.pyc", line 143, in get
  NameError: global name 'body' is not defined
```

Looks like a simple paste error.

https://github.com/scalyr/scalyr-agent-2/blob/master/scalyr_agent/connection.py#L143

**Comment by weilliu** [comment]
Hi @andrewq,

This is a known issue and we are working on a fix. 

In the meantime, you should still be able to upgrade by downloading/installing agent msi directly. That should unblock you from upgrading to the latest version. 

Let us know if you have any questions.

**Comment by andrewq** [comment]
Yes, thanks - sorry about the alarmist title. This is just an issue with the wrapper script.

**Comment by echee2** [comment]
@andrewq, just letting you know that we fixed this bug in 2.0.49.

Sorry for any inconvenience and thank you for reporting it.
